espnet/nets/pytorch_backend/conformer/subsampling.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
This file is still redundant.

Reply:
Same as above, include some differences in activation functions and whether to use rel_pos.

Reply:
In the original paper, the Swish activation is not placed in frontend CNN layers. Also, relative positional encoding shouldn't appear here.

Reply:
@pengchengguo, could you add the above descriptions of how it is different from the original transformer subsampling in the code?